.storybook/preview-head.html

```diff
@@ -4,3 +4,16 @@
   href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap"
   rel="stylesheet"
 />
+<!-- Import EDD template js -->
+<script type="text/javascript" src="/js/libs/jquery.js"></script>
+<script type="text/javascript" src="/js/cagov.core.js"></script>
```

Comment:
@lomky Please let me know if you can think of any issues with including these in storybook.

Reply:
nothing specific jumps to mind